netbox/templates/utilities/templatetags/utilization_graph.html

```diff
@@ -1,4 +1,4 @@
-<div class="progress text-center">
+<div title="Used: {{ util_count }}&#013Available: {{ num_available }}" class="progress text-center">
```

Comment:
We should only be including the tooltip if the `util_count` and `num_available` context variables are present.